python/doc/examples/calibration/bayesian_calibration/plot_gibbs_simus.py

```diff
@@ -0,0 +1,582 @@
+#
+# If this point does not respect the constraints, then we simply project
+# each component unto the constrained space.
+#
+# Hence the following 1D problem remains to be solved:
+#
+# .. math::
+#
+#     \widehat\tau = \arg\max_{\tau\in[\tau_{\min};\tau_{\max}]} \mathcal N(\vect{Y}^{obs}|\vect{\mu}_n(\tau) \mat{X}; \tau^{-1} \mat{I}_n + \mat{Q}^{-1}).
+#
+# The optimal value of :math:`\vect{\theta}` is then given by:
+# :math:`\widehat{\vect{\theta}} = \vect{\mu}_n(\widehat\tau).`
+
+def mu_n(tau):
+    x = ot.Point(initial_state)
+    x[p] = tau
+    # trick: posterior conditional mean is computed by the link function
+    return py_link_function_theta(x)[:p]
+
+
+# optimization criterion
+def log_cond_tau_post(tau):
+    x = ot.Point(initial_state)
+    x[p] = tau
+    # replace theta by its conditional posterior mean
+    x[:p] = py_link_function_theta(x)[:p]
+    # compute log conditional posterior of tau
+    ll = py_log_density(x)[0]
+    return ll
+
+
+# %%
+# 1D optimization
+func = lambda X: [-log_cond_tau_post(X[0])]
+problem = ot.OptimizationProblem(ot.PythonFunction(1, 1, func))
+problem.setBounds(ot.Interval([1e-4], [1e4]))
+solver = ot.TNC(problem)
+solver.setStartingPoint([1.0])
+solver.run()
+tauhat = solver.getResult().getOptimalPoint()[0]
+
+# inject result in initialState vector
+x = ot.Point(initial_state)
+x[:p] = mu_n(tauhat)
+x[p] = tauhat
+
+
+# %%
+# Final step : create a :class:`~openturns.MetropolisHastings` object for each block
+
+initialState = x
+
+mi_Y = [i for i in range(p + 1, p + 1 + n)]
+link_function_y = ot.PythonFunction(len(x), 4 * n, py_link_function_y)
+rvmh_Y = ot.RandomVectorMetropolisHastings(RV_Y, initialState, mi_Y, link_function_y)
+
+mi_theta = [i for i in range(p)]
+link_function_theta = ot.PythonFunction(len(x), p + (p + 2) * p, py_link_function_theta)
+rvmh_theta = ot.RandomVectorMetropolisHastings(
+    RV_theta, initialState, mi_theta, link_function_theta
+)
+
+log_pdf_tau = ot.PythonFunction(len(x), 1, py_log_density)
+rwmh_tau = ot.RandomWalkMetropolisHastings(
+    log_pdf_tau, support, initialState, proposal, [p]
+)
+
+# Now, assemble the blocks to create a Gibbs algorithm:
+gibbs = ot.Gibbs([rvmh_Y, rvmh_theta, rwmh_tau])
+
+# %%
+# Launch Algorithm
+sampleSize = 1000
+
+# Joint posterior density sample
+sample = gibbs.getSample(sampleSize)
+
+# compute acceptance rate
+tau_post = np.array(sample[:, p]).ravel()
+acc_rate = (tau_post[1:] != tau_post[:-1]).mean()
+print("Acceptance rate: %s" % acc_rate)
+
+
+# %%
+# Plot posterior distribution marginals
+
+# extract interest parameters
+post_sample = sample.getMarginal([i for i in range(p + 1)])
+post_sample.setDescription(["$\\theta_1$", "$\\theta_2$", "$\\tau$"])
+
+ks = ot.KernelSmoothing()
+posterior = ks.build(post_sample)
+
+# Optional: change the bandwidth
+bandwidth = ks.getBandwidth()
+for k in range(len(bandwidth)):
+    bandwidth[k] = 1.0 * bandwidth[k]
```

Comment:
I suggest this comment so as the content is as clear as possible. 
```suggestion
    bandwidth[k] = 1.0 * bandwidth[k]  # Change 1.0 to 2.0 to smooth the fitted distribution
```

Reply:
Good idea! Could you change your suggestion to "Change 1.0 to 2.0 to make the fitted distribution smoother". I don't think "smooth" can be used as a verb.

Reply:
I agree: this should be simply deleted.